references/sdk.md

```diff
@@ -161,6 +173,8 @@ Create new algo asset.
 }
 ```
 
+If an algo with the same file already exists, an `AlreadyExists` exception will be raised.
```

Comment:
```suggestion
If an algo with the same archive file already exists, an `AlreadyExists` exception will be raised.
```

I'm afraid people could get confused between the archive file vs the algo script file (same for other algos). What do you think?

Reply:
Good point, I'll make the update (and for objective as well).

Reply:
Updated. As for the objective I was mistaken as it's the description that is relevant and not the archive.